index.html

```diff
@@ -27,10 +26,23 @@
       <nav class="menu">
         <ul>
           <li><a href="#" class="logo">Logo</a></li>
-          <li>
-            <a href="#"><img src="./images/union.svg" alt="Hamburger logo" /></a>
+        </ul>
+        <ul class="nav-menu" id="main-menu">
+          <li class="nav-items nav1">
+            <a href="#my-projects" class="nav-link">Portfolio</a>
+          </li>
+          <li class="nav-items">
+            <a href="#about-me-2" class="nav-link">About</a>
+          </li>
+          <li class="nav-items">
+            <a href="#my-contact-form-2" class="nav-link">Contact</a>
           </li>
         </ul>
```

Comment:
Hello @Tufahel , you have indeed done a great job with the mobile menu feature and I have checked and most of the requirements have been met 💪🏻! However, I have noticed that on desktop size the hamburger menu button is visible which shouldn't be the case.

- [ ] Please kindly fix the mentioned above behavior so that on larger screens the main menu appears instead 😺.

Reply:
Hi @Cathella, maybe you have mistakenly mentioned my name. This is not my repo nor my team partner repo.

Reply:
Hi @Tufahel, I am very sorry for the mention.